src/test/java/bogdanSandu/tests/Tema5/DemoQa/UpdateTextField.java

```diff
@@ -0,0 +1,38 @@
+package bogdanSandu.tests.Tema5.DemoQa;
+
+import bogdanSandu.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class UpdateTextField {
+
+    public static void main(String[] args) {
+
+        updateTextField();
+    }
+
+    public static void updateTextField() {
+        ChromeDriver driver = BrowserManager.createDriverAndGetPage();
+        driver.get("https://demoqa.com/text-box");
+        WebElement fullName = driver.findElement(By.id("userName"));
+        fullName.click();
+        fullName.clear();
+        fullName.sendKeys("Bob ");
+        WebElement email = driver.findElement(By.id("userEmail"));
+        email.click();
+        email.clear();
+        email.sendKeys("adresamail@gmail.com");
+        WebElement currentAddress = driver.findElement(By.id("currentAddress"));
+        currentAddress.click();
+        currentAddress.clear();
+        currentAddress.sendKeys("Str. No.");
+        WebElement permanentAddress = driver.findElement(By.id("permanentAddress"));
+        permanentAddress.click();
```

Comment:
Try to scroll to this element before interacting with it. There are sometimes adds displayed in the page that covers this element and make the test to fail ->
Exception in thread "main" org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <textarea rows="5" cols="20" id="permanentAddress" class="form-control"></textarea> is not clickable at point (572, 612). Other element would receive the click: <iframe id="google_ads_iframe_/21849154601,22343295815/Ad.Plus-Anchor_0" name="google_ads_iframe_/21849154601,22343295815/Ad.Plus-Anchor_0" title="3rd party ad content" width="970" height="100" scrolling="no" marginwidth="0" marginheight="0" frameborder="0" role="region" aria-label="Advertisement" tabindex="0" sandbox="allow-forms allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts allow-top-navigation-by-user-activation" style="border: 0px; vertical-align: bottom;" data-load-complete="true" data-google-container-id="1"></iframe>